.github/workflows/cloud_regression.yml

```diff
@@ -52,3 +52,18 @@ jobs:
                "custom_netdata_image": "true"
                }'
           wait-for-completion: false
+
+      - name: Trigger Agent Parent/Child with Cloud Integration tests
+        uses: aurelien-baudet/workflow-dispatch@v2
+        with:
+          repo: netdata/test-automation
+          ref: refs/heads/master
+          workflow: agent_smoke_tests.yml
+          token: ${{ secrets.NETDATABOT_GITHUB_TOKEN  }}
+          inputs: '{ "netdata_branch": "${{ steps.output-workflow-dispatch-params.outputs.netdata_branch }}",
+               "netdata_repo": "${{ steps.output-workflow-dispatch-params.outputs.netdata_repo }}",
+               "netdata_pr_number": "${{ steps.output-workflow-dispatch-params.outputs.netdata_pr_number }}",
+               "netdata_branch_commit_hash": "${{ steps.output-workflow-dispatch-params.outputs.netdata_commit_hash }}",
+               "custom_netdata_image": "true"
+               }'
+          wait-for-completion: true
```

Comment:
Should we add a blank line at the end @Ferroin ?

Reply:
In theory yes, but in practice it really doesn’t matter much here.

Reply:
Isn't there one? My IDE shows it as having one at the end 🤔

Reply:
Both yamllint and the diff here on GitHub do not show one.

Reply:
@dimko , please , add one for us.

Reply:
@thiagoftsm 50$ 😛

Reply:
I will consider that you are asking for Uruguayan pesos :smile: